webpack.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'main.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$/i,
+        use: ['style-loader', 'css-loader', 'sass-loader'],
+      },
+    ],
```

Comment:
- [ ] The project fails to compile with the error: `Error: Can't resolve '\weather-app\src\
  index.html'`. You could either update the directory structure and place index.html in `src` directory or update the ` webpack.config`.